.github/workflows/main.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+
+on: 
+  push:
+    branches:
+      - master
+# For scheduled deployment with Cron Jobs.
+## Examples of cron schedule expressions:
+### '0 * * * *': hourly
+### '0 0 * * *': daily
+### '0 0 1 * *': monthly
+#   schedule:
+#     - cron:  '0 0 * * *'
+
+jobs:
+  deploy:
+
```

Comment:
Extra line:

```suggestion
deploy:
    runs-on: ubuntu-latest
```

Reply:
Thanks for spotting that and I have fixed it.